website/www/site/layouts/_default/list.rss.xml

```diff
@@ -0,0 +1,50 @@
+{{/*
```

Comment:
Please add link in the footer

Reply:
Also, I think the url should be /feed.xml, not /index.xml, so that it's compatible with old links

Reply:
Add license pls

Reply:
I see that we're missing category :( Do you think we should add that?